package.json

```diff
@@ -1,16 +0,0 @@
-{
-  "name": "restapi",
-  "version": "1.0.0",
```

Comment:
How do we use versions?
How do we use Git to manage multiple versions?
How do we use git versions so we don't break changes on production?